recipes/_generate_password.rb

```diff
@@ -8,6 +8,8 @@
 
 return if node['splunk']['free_license'] && node['splunk']['node_type'] != :forwarder
 
+return if node['splunk']['ignore_already_installed_instance'] == true
```

Comment:
why are we skipping this step?  Under normal circumstances we do this for both UFs and full package installs.

Reply:
It caused issue with the other instance of splunk that is running. It would change the password on file but not actually change the password running in splunk. 

Now that I think about it we would have the same issue when a normal chef would be ran and then the universalforwarder would have a wrong password. I'll have to change this up.

Reply:
https://github.com/cerner/cerner_splunk/pull/265/commits/2446ced75d2da233b1901d9109f92917bce8cf1f

This should fix this issue. Gist of tests https://gist.github.com/HCarr64/2e7fbc6f22b51157d457d133e77b043e